analytics.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/proto/livekit";
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+service AnalyticsRecorderService {
+  rpc IngestStats(stream  AnalyticsStats) returns (google.protobuf.Empty){};
+  rpc IngestEvents(stream AnalyticsEvents) returns (google.protobuf.Empty){};
+}
+
+enum STATSSTREAMTYPE {
```

Comment:
I think for consistency sake, let's keep these to be CamelCase. same as enum values. I would just call this `StreamType`